config/media.php

```diff
@@ -27,19 +27,83 @@
         ],
     ],
 
-    /*
-     * user for news, ...
+    /**
+     * used for news, ...
      */
     'header' => [
         '2xl' => [
 
         ],
     ],
 
-    /*
-     * user for games
+    /**
+     * used for game-related media
      * TODO: aspect ratio should be defined by system
      */
     'game' => [
+        /**
+         * banner images
+         * game editors upload 32:9 source images
+         * mobile uses 16:9 crops, desktop uses full 32:9
+         * uses singleFile() - only keeps the latest version to minimize storage costs
+         */
+        'banner' => [
+            'mobile-sm' => [
+                'width' => 640,
+                'height' => 360,
+            ],
+            'mobile-md' => [
+                'width' => 1024,
+                'height' => 576,
+            ],
+            'desktop-md' => [
+                'width' => 1024,
+                'height' => 288,
+            ],
+            'desktop-lg' => [
+                'width' => 1280,
+                'height' => 360,
+            ],
+            'desktop-xl' => [
+                'width' => 1920,
+                'height' => 540,
+            ],
+        ],
+
+        // Future game media types can go here:
+        // 'screenshot' => [...],
+        // 'boxart' => [...],
+    ],
+
+    /**
+     * used for user-related media
+     */
+    'user' => [
+    /**
+     * profile banners (future feature)
+     * given a user beats a game that has a custom banner, users can use that banner on their profile
+     * when user selects a game banner, a COPY is stored in the profile_banner library as a snapshot
+     * this ensures their profile banner doesn't change if the game's banner is updated
+     * uses the same sizes as game.banner for consistency
+     * does NOT use singleFile() - users can save multiple banners
+     *
+     * implementation approach:
+     * 1. user beats a game -> unlocks ability to use that game's banner
+     * 2. user selects the banner -> copy media from game.banner to user.profile_banner
+     * 3. store source game_id in custom properties for attribution
+     * 4. user can collect multiple banners (1 per beaten game)
+     * 5. user selects active banner via some preference setting
+     *
+     * storage considerations:
+     * - game banners remain on singleFile() for S3 disk efficiency
+     * - user profile banners are stable independent snapshots
+     */
+        // 'profile_banner' => [
+        //     'mobile-sm' => [...], // same sizes as game.banner
+        //     'mobile-md' => [...],
+        //     'desktop-md' => [...],
+        //     'desktop-lg' => [...],
+        //     'desktop-xl' => [...],
+        // ],
```

Comment:
This is documenting an idea for future functionality.

I think it would be neat if, once user profiles are migrated to React, users could (optionally) set their profile banner image to the same banner image of a game they've beaten. This has an added benefit of further incentivizing beaten awards and making profiles a bit more aesthetically pleasing.

We can figure out the technical details later - I wanted to dump all the thoughts from my brain in here before I lost them.

Reply:
Removed this whole blurb in latest.